closed/adds/jdk/src/share/native/jdk/crypto/jniprovider/NativeCrypto.c

```diff
@@ -717,36 +984,31 @@ JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_GCMDecrypt
             (*env)->ReleasePrimitiveArrayCritical(env, key, keyNative, JNI_ABORT);
             (*env)->ReleasePrimitiveArrayCritical(env, iv, ivNative, JNI_ABORT);
             (*env)->ReleasePrimitiveArrayCritical(env, output, outputNative, JNI_ABORT);
-            if (inLen > 0) {
+            if (inLen > 0)
```

Comment:
Please restore the braces around the `if` body.

Reply:
Didn't think I took them out... Fixed anyway.